### ISSUE TITLE
Contexto e história do jogo

### ISSUE THREAD

**Issue by luisvmjr** [luisvmjr/JogoAPDA#1]
Bem, todo jogo tem que ter enredo e contexto como se fosse uma história mesmo. Então, para quem não tem capacidade de estar ajudando tanto com a programação pode estar ajudando aqui com ideias e futuramente com sprites / design.

**Comment by MagoDosCodigos** [comment]
um índio que luta contra os portugueses invasores?

**Comment by luisvmjr** [comment]
Não, HUAHUAUH, sim um MMORPG. Tema: A ser decidido ainda.

**Comment by gtnardy** [comment]
Em minha opinião, vejo que seria interessante um jogo que mesclasse um Path of Exile, no quesito árvore de habilidade e customização de poderes; RuneScape, no quesito de coletar e craftar equipamentos e materiais, e em ter várias habilidades gerais como Magia, Defesa, e também de crafting, como Mineração e Pescaria, por exemplo; E talvez algo como Rust/Minecraft nos quais podemos construir casas ou bases. Se tudo conseguisse ser miscigenada em uma jogatina em primeira pessoa, ficaria bem massa.

**Comment by Marcos-Oleiro** [comment]
Uma coisa que eu tive no Tibia e até agora eu não achei em outro jogo é a não-obrigatoriedade de fazer quests (só para certos itens e acesso aos locais). Eu podia logar e ficar caçando horas e horas. Acho que podíamos seguir isso também, ou algo similar.

**Comment by MagoDosCodigos** [comment]
Gostei da ideia, marcos.

Em 24 de dezembro de 2015 20:31, Marcos-Oleiro <notifications@github.com>
escreveu:

> Uma coisa que eu tive no Tibia e até agora eu não achei em outro jogo é a
> não-obrigatoriedade de fazer quests (só para certos itens e acesso aos
> locais). Eu podia logar e ficar caçando horas e horas. Acho que podíamos
> seguir isso também, ou algo similar.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/luisvmjr/JogoAPDA/issues/1#issuecomment-167163917>.
>



-- 
*Tiago Carneiro Vizoto*
*Cel:* (041) 9278-2762
*Skype:* @tiagovizoto
*Freelancer Back-End *
(Python, Ruby on Rails e Java)

**Comment by datafanatic** [comment]
Gostei da ideia de ambos @Marcos-Oleiro e @gtnardy. Em relação à ideia do Marcos, há alguma ideia de como implementar? Eu joguei Tibia, mas foi bem pouco e nunca levei a serio. Poderia me explicar o sistema? Todos os itens eram conseguidos com loot de monstros e jogadores? E em relação à experiência e nível do personagem?

**Comment by Marcos-Oleiro** [comment]
No Tibia a experiência era obtida somente com a mortes de monstros. Eram espalhados no mundo todo e em áreas. Em relação aos equipamentos, a maioria poderia ser obtida de monstros (os melhores eram mais raros ou caim de criaturas que necessita um nível alto) mas alguns eram somente obtidos por quests ( tipo a soft boots, rainbow shield, SoV e etc). Os acessos a alguns lugares eram obtidos somente com quests, mas a grande maioria o acesso era direto.O que impedia o acesso era a dificuldade dos monstros (alta HP, alto dano).

P.S : Saudades SoV, sdds.

**Comment by datafanatic** [comment]
Entendi. Então, seria algo tipo um monstro(s) forte(s) (hp + dano) na frente de uma caverna e após o jogador matar tais monstros ele teria acesso à uma nova área com monstros mais fortes? Ou só a caverna sem nenhum monstro "guardando" ela?

**Comment by Marcos-Oleiro** [comment]
Não, seria tipo assim: um lvl 30 poderia chegar numa 'cave' de demons mas ele ia levar um hit dele e ia morrer. O que limitava era os monstros no lugar ou até mesmo no trajeto. Mas nada impedia que alguém com nível maior ajudasse esse nível 30 a matar o demon (tankando o demon pra ele).

**Comment by datafanatic** [comment]
Certo. E a divisão de XP? Era igual para todos os jogadores da party ou os de nível maior levavam menos e os de nível menor levavam mais?

**Comment by Marcos-Oleiro** [comment]
Se a diferença dos níveis dos membros fosse dentro do limita que a CIP estipulou a xp era dividida igualmente, senão era dividida de acordo com o dano causado ao monstro acho.

**Comment by gtnardy** [comment]
Em um jogo que conheço (MapleStory), a exp era dividida entre todos membros da party, mas dada apenas ao que causaram dano significativo ao monstro.

**Comment by gtnardy** [comment]
Como os mais novatos não têm dano para isso, acabam não recebendo exp.

**Comment by datafanatic** [comment]
@Marcos-Oleiro e @gtnardy 


Por favor, avaliem:

Membros da party: 2 jogadores com níveis de 40 e 50, 3 jogadores com níveis de 10, 23, 25.
Dano médio dos jogadores de nível alto: 100~200.
Dano médio dos jogadores de nível baixo: 5~50.
Monstro nível 30:
- HP: 4200
- Experiência: 5000

Divisão de experiência:
- (5000/5) * 1.5 pra cada jogador;

ou
- x: 5000 - Experiência total do monstro;
- y: 5 - Número de membros da party;
- z: 30 - Nível do jogador multiplicado pelo número de membros da party.
- (x/y) - z * y

**Comment by luisvmjr** [comment]
Honestamente, eu acho que a experiência deverias ser sempre dividida de forma igual entre os membros. Independente do nível. Claro, desde que causem ao menos dano ao "monstro". 
Uma vez que isto facilitará jogadores mais antigos a ajudar a "upar" os jogadores mais novos e vice e ver. E quem nunca teve um char que quis upar mais rápido? Mas ficava limitado a essas regras. 
Então apoio tal sistema 
Divisão de experiência:

    (5000/5) * 1.5 pra cada jogador;
caso haja + do que 5 jogadores em uma pary:
 (y/x) * 2.0 pra cada jogador;

Onde y = exp ; x = membros. Estimularíamos o jogo em equipe :)

**Comment by Marcos-Oleiro** [comment]
Acho que a soma das experiência de cada jogador da party não pode ultrapassar a experiência total do monstro. Na primeira sugestão cada jogador iria ganhar 1500 de xp, somando mais que 5000. E no segundo caso, acho que somariam menos que a xp total né?
No caso de querer upar um char lvl baixo, é não entrar em party (o que era feito no tibia).O que eu acho perigoso nessa 'facilidade' de upar um lvl baixo tendo um lvl alto amigo é que no caso dum grupo consiga o 'controle' de uma área sem deixar mais ninguém usar a área, vai ser facilitado por isso, o que atrapalharia os outros jogadores pq o tal grupo de level alto teria mais facilidade de upar lvls baixos e ganharia mais poder mais facilmente, o que, dependendo do tipo de jogador, pode ser prejudicial.

**Comment by luisvmjr** [comment]
Claro, e o que vocês pensam em **vocações**?
Devemos lembrar que devem ser balanceadas, e **"justas"** no **pvp**.

##Ambientação do jogo
Penso em dar aos jogadores, uma terra totalmente descolonizada, sem nada, o começo de um mundo, e possibilitar o avanço da mesma, entendem? como civilização, até a idade média. Limitar o MÁXIMO o numero de npc's para manter a interação de jogadores, precisam de comidas? procurem os fazendeiros. Precisam de armaduras e armas? Procurem os ferreiros. Assim, além de caça de monstro abrimos um rpg diferente, onde você pode ser uma profissão :)

**Comment by datafanatic** [comment]
É interessante, mas eu vejo um problema aí. Isso desestimularia novos jogadores a começarem no jogo pois iriam evoluir muito lentamente em comparação com aqueles que já têm conhecidos de nível alto no jogo. E eu acho que até certo ponto ficaria fácil demais. Imagina só:

Membros da party: 2
- Um jogador nível 90;
- Um jogador nível 30;
Dano dos jogadores:
- 400~750;
- 100~150;
Monstro nível 50:
- HP: 7500
- Experiência: 11000

Experiência para os jogadores (usando sua fórmula):
(11000/2) * 2.0 = 11000 (?)
Não faz sentido pois os jogadores de nível baixo iriam subir MUITO rápido de nível.

@Marcos-Oleiro Olha, ali os números passam de 5000 por causa do bônus (tanto o de 2.0 quanto o de 1.5) esse bônus não seriam permanentes e só seriam dados caso a discrepância de níveis fosse pequena ou houvesse algum tipo de buff dado por algum jogador da party. Concordo em relação ao controle da área, isso é bem inconveniente e é terrível tentar ir farmar num lugar onde alguém "tomou controle". Um jeito de se resolver isso seria usando um sistema de áreas controladas, exemplo:

Grupo 1 tomou controle do Castelo XYZ. Monte seu grupo e venha retomar o controle do Castelo e receba prêmios!

E desencorajar jogadores de "tomarem controle" de qualquer outra área senão as controladas. O que acha?

**Comment by Marcos-Oleiro** [comment]
@datafanatic o bonus por party com leveis semelhantes até faz sentido sim.

Quanto ao controle de área, a ideia de áreas que são feitas pra serem disputadas é uma boa sim. Mas quanto às áreas que são, à princípio, livres acho que só é possível com controle in-game acho, punição mesmo.Fazer um jogo totalmente sandbox como o Tibia diz ser não dá certo acho, tem que ter leis de convívio, leis de respeito, senão vira o que virou o Tibia atualemente.

**Comment by Marcos-Oleiro** [comment]
@luisvmjr 

No curso que eu tô fazendo de Game Design eu pensei em fazer 6 classes: tank, healer, 2 dps, DPS/Tank e DPS/healer.

Quanto ao ambiente pode ser ou um 'novo mundo' ou a missão de reerguer um mundo destruído por alguma coisa, guerra, invasão dos dômônios, alguma catástrofe.

**Comment by luisvmjr** [comment]
@Marcos-Oleiro  Acho tão legal a ideia de um mundo do zero onde "temos" que construir tudo, mas que haja sim monstros misticos, dragões etc. O que é DPS? O.o

**Comment by Marcos-Oleiro** [comment]
Damage per Second, é a classe danosa xD

**Comment by luisvmjr** [comment]
E se nós fizéssemos um rpg sem classes? Que o que definia o que tu fosse seria teus equipamentos e "atributos" que escolhesse, como no path of exile? Seria **foda**.
Aí depende do jogadores escolher, pode fazer full tank, full dano, depende de sua build :+1:

**Comment by datafanatic** [comment]
@Marcos-Oleiro Eu consigo pensar em 2 jeitos:

O primeiro é como você falou controle in-game forte e com punho de ferro

Prós:
- Funcionalidade quase 100%;
- Sistema de punições ao vivo;

Cons.:

- Precisa de um Game Master quase 24/7 no jogo;

Método automático:

O jogador é expulso de área X se tiver nível muito alto comparado com os monstros da área e ficar lá por muito tempo (seja de movimentando ou IDLE).

@luisvmjr Eu gostei da ideia. Na verdade, achei sensacional. Tive, porém, problemas pensando numa possível implementação justa. Quer dizer, como os primeiros ferreiros conseguiriam minérios pra fazer as armaduras/armas? E os primeiros mercadores, de onde conseguiriam mercadorias pra vender em outra cidade/área? O que eu pensei aqui seria assim:

Começamos o mundo com uma quantidade aceitável de (insira qualquer vocação aqui, exemplo, ferreiro):

- 50 Ferreiros NPC pelo mundo
- 15 Jogadores com vocação de Ferreiro

Um mês depois:

- 30 Ferreiros pelo mundo
- 40 jogadores com vocação de Ferreiro

Deu pra entender a ideia? Os NPCs seriam retirados do jogo conforme o mesmo fosse evoluindo economicamente até se tornar auto-sustentável entre os próprios jogadores.

**Comment by gtnardy** [comment]
Sem classes é legal, pois o jogador pode, a qualquer momento, alternar ou começar um modo de jogo diferente, sem ter que começar do 0 com uma 'classe' nova.

**Comment by Marcos-Oleiro** [comment]
@datafanatic 
A questão de tempo do jogador na área seria facilmente burlada por revezamento de players/personagens. Acho que infelizmente a melhor coisa seria mesmo in-game embora tenha esse contra. Eu não consigo pensar em uma maneira automática de controlar isso.

**Comment by gtnardy** [comment]
@datafanatic, talvez esses NPCs poderiam ser repensados, no lugar de NPCs que fazem/vendem  itens/equipamentos, poderia deixar lugares (ex. fornalhas) em que o jogador possa forjar itens conforme seu nível em 'forjamento', quanto maior o nível, mais itens melhores com mais diversos minérios poderão ser forjados, ideia similar ao RuneScape.

**Comment by luisvmjr** [comment]
@datafanatic 
Sim, substituiriamos os npc's pelos próprios jogadores.... Perfeito.

# Sobre atributos:

No Tibia o que eu mais gostava e em outros jogos é que tu podia ter "Atributos" (skills), já que por sua vez eram obtidos através de "treino", isso é perfeito, entendem? Já que teriamos diferentes profissões, suas habilidades seriam implementas quando as treinassem. Se um jogador decidiu ser um escudeiro, deveria treinar sua habilidade de bloquear dano, com outro jogador que o atacasse. Um ferreiro bom é aquele que já produziu varias armas, e assim elas vão aumentando de qualidade. Entendem o que eu quero dizer? Possibilitando assim um jogo diferente dos demais, fazendo com que seja por merecer o que você tem, não só por "UPAR" e adicionar o seu atributo, 31903281 jogos já fazem isso, porque mais um igual? :+1:

**Comment by Marcos-Oleiro** [comment]
@luisvmjr 
Mas ae o cara que quisesse jogar de tank seria obrigado a 'treinar' o atributo 'ferreiro'? Ou ele poderia comprar escudos ou alguma coisa que o ajudasse de algum jogador com skill de 'ferreiro' bom?

**Comment by dielsojorge** [comment]
Poderíamos usar o mapa múndi para ser a base do game? As habilidades e personagem e forma de enriquecimento viriam do país ou região escolhida.Jogadores poderiam criar suas fabricas, fazendas, exercito para vender os seus servições de luta, ser minerador. Poderia ter negociação entre jogadores, pagamentos por proteger minha base/casa/cidade.

**Comment by dielsojorge** [comment]
Não vamos nos prender apenas a luta ganho de experiencia, atributos poderiam ser dano magico e físico, velocidade de movimento, velocidade de ataque, vida do herói, e defesa. Quanto mais simples melhor. O jogador decidiria o que fazer com seu personagem, se iria lutar e virar guerreiro, se seria um comerciante, fazendeiro, mineiro, curandeiro, Podendo mudar de profissão o momento que quisesse. Até por que Existe jogador que sabe negociar, outros lutar etc.

**Comment by datafanatic** [comment]
@Marcos-Oleiro Pensei em outra alternativa.

Áreas com grande concentração de jogadores de nível baixo seriam proibidas para jogadores de nível (muito) mais alto.

@gtnardy Isso é bom e faz bastante sentido. No caso, qualquer jogador poderia fazer os próprios equipamentos, mas só jogadores com vocação de ferreiro poderiam fazer e vender itens de nível e proficiência alta.

@luisvmjr Nenhuma objeção em relação ao treino.

@dielsojorge É muito grande, fica inviável pra um MMORPG, mas a ideia de habilidades relacionadas à cultura da região de onde o personagem vem é ótima. Gostei bastante.

**Comment by Marcos-Oleiro** [comment]
@datafanatic 
É, eu sempre achei que um level limite para área seria algo muito útil para evitar o abuso.

**Comment by luisvmjr** [comment]
@dielsojorge as formulas de defsa seriam tipo:
DanoDoMonstro - (DefesaDoEscudo * HabilidaDeDefesa) = DanoRecebido :)

Todos que "treinassem" poderiam ser tudo, mas também dependem de seus itens, e os itens também limitam que um tanker seja full dano, se tivesse treinado os dois, porque armas de alto dano seriam por exemplo, machados de duas mãos, o que ficaria inviável com o uso de um escudo. 
----------------------------------------------------------------------------
Agora sobre o mapa mundi, não curti. Prefiro que jogadores formem suas vilas e vilarejos a partir das afinidades que sentirem um com os outros, não estipular tantas limitações. Mas de parte de habilidades relacionadas a região em que vive, faz todo sentido, alguém que vive no deserto não poderá ter tanta habilidade de "fazendeiro" quanto alguém que vive em uma área agrícola. Diferentes biomas, diferentes riquezas, diferentes fontes de dinheiro e experiência.

**Comment by dielsojorge** [comment]
@datafanatic Não usaríamos a área real do mundo em que vivemos, diminuiríamos bastante a escala. O mapa seria apenas o molde, seriam apenas os continentes.

**Comment by edsonsouza92** [comment]
@luisvmjr Essa parte de jogadores formarem suas vilas, ter habilidades de "fazendeiro", por exemplo, além de também "mineração" e/ou outros tipos de cultivos, você está pensando em algo estilo sandbox?

**Comment by luisvmjr** [comment]
@edsonsouza92 sim, tudo seria construído por jogadores.

**Comment by edsonsouza92** [comment]
@luisvmjr gostei bastante dessa idéia. 
Sobre a discussão das classes, acho que é interessante ter classes no jogo. Pelo menos os nomes delas, mas ela seria atribuida ao jogador dependendo dos atributos que ele treinou. Ou seja, ele construia sua própria classe, mas o nome delas já estaria pré-definido.

Por exemplo. Se ele treinou muita agi / str... Então já teria 5 classes possíveis que ele poderia ser, caso treinasse outro atributo diminuiria a quantidade de classes ou mudaria completamente essas classes dependendo de quanto ele treinou esse novo atributo.

**Comment by MagoDosCodigos** [comment]
Tendo a classe mago, está ótimo.

**Comment by luisvmjr** [comment]
@MagoDosCodigos Rótulos são ruins, :v:

**Comment by datafanatic** [comment]
@Marcos-Oleiro E não haveria limitação pra quem fosse nível baixo e fosse pra área alta, ou seja, se teu amigo nível alto quisesse te ajudar a subir de nível, você iria pra uma área de nível alto e não vice-versa. Eu acho justo.

@luisvmjr Concordo que grande parte das estruturas precisam ser construídas pelos jogadores, mas nós (os desenvolvedores) precisamos de um mapa base pra trabalhar em cima. Vai tomar MUITO tempo se nós mesmos formos construir as coisas pra então testar o funcionamento.

**Comment by deoprog** [comment]
seria estranho um teste de aptidão do jogador e com isso,  avaliar o primeiro nível do mesmo? um teste ou vários teste por nível que daria uma graduação ao jogador, algo como análise da expertiasia do player.!

**Comment by datafanatic** [comment]
A propósito, muitas das sugestões dadas aqui apesar de serem boas, deixariam o jogo muito pesado e, consequentemente, inviável pra ser rodado no navegador (leia-se javascript). Eu acho que deveríamos começar a considerar usar C++ (Cocos-2d, SFML, Allegro).

@deoprog Não entendi sua pergunta. Nível de aptidão relacionado a o que? Ao jogo? MMORPG? Classe? Vocação?

**Comment by luisvmjr** [comment]
@datafanatic Lucas Araújo, tu é parente do Apolo Araújo?
Em minha opinião, ainda vamos escrever nossa própria engine do jeito que está, para satisfazer todas as necessidades. E deixariamos ela open-source também.

**Comment by dielsojorge** [comment]
Essas classes que já vem sendo usadas não são boas, classes devem ser guerreiro(Esse pode ser arqueiro, espada, magias,), comerciante(ferreiro, minerador, fazendeiro, curandeiro.)

**Comment by dielsojorge** [comment]
@datafanatic Também prefiro em C++

**Comment by luisvmjr** [comment]
@dielsojorge o problema é rotular, não precisa ser rotulado o que cada um é, não tem necessidade. Acho legal a pessoa poder der mais de uma coisa, só que demoraria muito tempo para a pessoa se especializar em varias áreas. Guerreiro / cozinheiro e curandeiro por exemplo. Possibilitamos a todos tudo, e sem precisar ficar criando personagens novos a toda hora.

**Comment by datafanatic** [comment]
@luisvmjr Depende. Ele é do Nordeste ou tem antepassados do Nordeste? Pois se for, é possível que sim.
Olha, eu acho desnecessário a gente criar nossa própria engine só pra esse jogo. O que seria ideal, ao meu ver, seria dar fork no Phaser e modificar pro nosso gosto.

**Comment by dielsojorge** [comment]
@luisvmjr O cara não vai escolher quando for criar o personagem a sua classe, o que digo é que ele pode seguir os caminhos que sugeri, comercio, luta etc. Entende.

**Comment by dielsojorge** [comment]
No próprio jogo ele decide.

**Comment by luisvmjr** [comment]
@datafanatic Não sei. Negócio é criar um capacete igual ao do SAO e GG! Ok, voltamos ao mundo real.
É o jeito é mudar o phaser e trazer a maior parte do processamento para o back-end e usar minified js, para diminuir o peso do jogo. Bem, teremos que adotar muitas técnicas para deixar o jogo leve.
Estarei criando uma issue chamada Sugestões técnicas. Discutimos o restante lá.

@dielsojorge Sim, entendo.

**Comment by jhonatanwestphal** [comment]
Galera, eu achei massa todas as ideias aqui, deixo como sugestão o seguinte:

1. Acho muito boa a ideia de que os players sejam livres a mudar de vocação pelo próprio treino, assim como pode usar bem o arco e a espada. Isso é realmente interessante.
2. Acho que com esta liberdade pode ser estimulado o comércio dentro do jogo. Compra-se as ferramentas de forja e cada um pode forjar seus equipamentos de acordo com o nível de forja, entretanto algo que tinha no Tibia também era a venda de "lojinhas", onde caso o player queira, pode simplesmente ficar vendendo.
3. Com isso a mineração, e coleta de outros materiais podem ser vendidos a quem quer forjar, assim como a quem quer fazer poções/runas. 
4. Os bichos não precisam dropar moedas, itens e afins. Eles podem sim serem utilizados para a coleta de marfim, couros, patas, molares e afins, utilizando também à forja de itens.
5. Não precisam ter itens pré-estabelecidos, pode sempre, baseado nos itens utilizados, forjar combinações de nomes com defesas/ataques baseados nos atributos dos insumos.
6. Não haveria level, e a territorialidade seria estimulada nas cidades, o que poderia conquistar seriam títulos de nobreza/bravura, o que seria diferente da maioria dos RPGs.
7. As quests podem ser complementos, baseados na necessidade do jogo. Aos poucos as quests são criadas, para domínio de novos territórios, aprendizado de uma nova linguagem, conquista de aliança com determinado povo ou até mesmo rivalidade.
8. Explorar não a caça, mas sim a história que cada personagem possui. Com quests "especiais" pode ser mantido o título de quem completou determina quest, resolveu determinado desafio, criando legados dentro do jogo.

Acho que RPGs de level já estão muito massantes, mas RPGs que interagem a história do personagem conjunto a história do jogo, isso seria inovador.

**Comment by Marcos-Oleiro** [comment]
@jhonatanwestphal 
Acho que mantendo o interesse pelo player pela 'hunt', como tem no tibia, pode ser feito isso que tu falou. A 'hunt' dava lucro/xp, além da baita diversão (pra mim pelo menos). E eu acho que o nível tem que existir sim e a xp só possa ser obtida dos monstros mesmo. E caso o player não queira trenar nada de profissão e tals, ele possa comprar de outro players.

**Comment by datafanatic** [comment]
1. De fato.
2. Sim, concordo. O que eu tinha em mente, porém, seria qualquer um poderia forjar equipamentos dependendo do nível da vocação de ferreiro, mas só aqueles que adotaram ferreiro como vocação principal podem forjar os equipamentos raros (ou muito poderosos) e podem vender os equipamentos. Os outros jogadores teriam que trocar (item por item, leia-se) ao invés de vender.
3. Sim, mas pra isso acontecer há de haver estimulo para os jogadores venderem minérios aos ferreiros ao invés de guardarem na stash.
4. 
5. Soa interessante, mas e jogadores mais novos que ainda não conseguiram a primeira vocação ou simplesmente não sabem forjar?
6. Fiquei com pensamentos mistos aqui. E o nível minimo para os equipamentos? Como seria conseguida a nobreza/bravura? Que critérios usar?
7. Gostei também.
8. Deixa eu ver se eu entendi, certas quests ficariam com o nome do jogador que completou elas em tempo recorde ou diferencialmente de outros que tentaram? Exemplo: "jhonatanwestphal foi um nobre {insira sua classe} que derrotou o Basilisco após 9 jogadores falharem sem completar o feito. A missão durou de x/x/xxxx até x/x/xxxx"? Seria algo assim?

Eu concordo que o sistema de níveis é massante, mas e se diferentemente de outros RPGs nós não dessemos ênfase ao nível e deixar ele só como um sistema de limitação, organização e divisão de XP como comentamos [aqui](https://github.com/luisvmjr/JogoAPDA/issues/1#issuecomment-167166284)?

**Comment by jhonatanwestphal** [comment]
@Marcos-Oleiro a caça seria mantida para pegar insumos e vendê-los/utilizá-los.

@datafanatic 

2. A intenção era não focar na vocação mesmo, todos podem ser tudo, mas claro, tudo tem uma certa dificuldade para evoluir e é baseado nisso que serão criados os itens. O nível de forja poderia diminuir o tempo utilizado para forjar e as chances de quebrar.
3. O estímulo é justamente o poder de troca. Quando não há gold caindo em monstros, automaticamente todo item conquistado em uma caçada é convertido em dinheiro, através da venda dos insumos.
5. Mantendo a ideia de sem vocação, há também os monstros fáceis. Tais monstros também dropam insumos, que podem ser vendidos para a compra de alguma armadura, ou então, a troca de insumos por determinada armadura (mais uma vez estimulando o comércio). Para que sempre haja venda, pode haver NPCs, que paguem uma quantidade bem baixa nos insumos, o suficiente para não valer a pena, mas "quebrar um galho" caso precise.
6. Sem nível mínimo para armaduras, talvez, a utilização de "treino mínimo", mas nada de leveis. A nobreza e bravura poderia ser conquistado através do domínio de territorialidade, áreas encontradas, quests finalizadas, legados atingidos, mas como algo de segundo plano, talvez com algumas vantagens.
7. Não, quests únicas mesmo. Conforme as quests são criadas baseadas na necessidade, o player ganha seu achievement e o jogo ganha um nome à história, podendo ser agregado a livros dentro do jogo, a fala de NPC's, a novas quests, venerações de povos, comentários em vilas, enfim.

PS: como o foco seria mantido no comércio, as próprias quests seriam formadas pelos comerciantes. "Colete 100 pedaços de madeira, em troca dou-lhe uma armadura, ouro, enfim".
Os comerciantes, geralmente, terão poder de luta menor, visto que não estão em constante treino. Desta forma, não conseguem finalizar quests mais "pesadas" e pode haver troca aí também.

**Comment by luisvmjr** [comment]
Estarei hoje a noite, separando as ideias que foram aprovadas e fazendo um PDF e efetuando o UPLOAD do mesmo arquivo. E estaremos organizando o começo de desenvolvimento orientado pelo Felipe. Infelizmente amanhã não participarei de modo algum na organização dos eventos a serem seguidos, estarei ocupado fazendo minha mudança, por isso peço que vocês colaborem e tentem ajudar a organizar o desenvolvimento também.

# Fim de tópico.
